hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -200,6 +199,14 @@ public static void createInflightCommitFiles(String basePath, String... instantT
     }
   }
 
+  public static HoodieWriteStat createReplaceStat(final String partitionPath, final String fileId1) {
```

Comment:
there is nothing specific about replace in this method? should we move this to the test class itself. inline?

Reply:
Done.